clusterloader2/pkg/framework/client/objects.go

```diff
@@ -99,6 +102,37 @@ func DeleteNamespace(c clientset.Interface, namespace string) error {
 	return RetryWithExponentialBackOff(retryFunction(deleteFunc, apierrs.IsNotFound))
 }
 
+// ListNamespaces returns list of existing namespace names.
+func ListNamespaces(c clientset.Interface) ([]string, error) {
+	var namespaceNames []string
+	namespacesList, err := c.CoreV1().Namespaces().List(metav1.ListOptions{})
+	if err != nil {
+		return namespaceNames, err
+	}
+	for _, namespace := range namespacesList.Items {
+		namespaceNames = append(namespaceNames, namespace.Name)
+	}
+	return namespaceNames, nil
+}
+
+// WaitForDeleteNamespace waits untils namespace is terminated.
+func WaitForDeleteNamespace(c clientset.Interface, namespace string) error {
+	retryWaitFunc := func() (bool, error) {
+		_, err := c.CoreV1().Namespaces().Get(namespace, metav1.GetOptions{})
+		if err != nil {
+			if apierrs.IsNotFound(err) {
+				return true, nil
+			}
+			if IsRetryableAPIError(err) {
```

Comment:
Could you negate this condition and remove the extra return statement?

Reply:
Done.